src/api/test/unit/code_quality_test.rb

```diff
@@ -77,7 +77,7 @@ def setup
       'BsRequestActionSubmit#execute_accept'                                    => 126.42,
       'BsRequestPermissionCheck#cmd_changestate_permissions'                    => 117.09,
       'RequestSourceDiff::ActionSourceDiffer#diff_for_source'                   => 94.62,
-      'BuildController#file'                                                    => 127.42,
+      'BuildController#file'                                                    => 130.44,
```

Comment:
I agree, I also don't like this test. We are ignoring it and I doubt that it can useful in some way. @openSUSE/open-build-service what do you think? can we remove it?